### ISSUE TITLE
Posthog <> Sentry exception tracking is not linking to the correct Sentry page

### ISSUE THREAD

**Issue by hexcowboy** [PostHog/posthog#23796]
## Bug description

We have the Postgres <> Sentry integration in our React app. It properly links the Sentry exception when one is captured in a session recording, but the link to the Sentry issue is invalid.

## How to reproduce

1. View a Session Recording that captured an Exception
2. Click on the Sentry external link 
  ![image](https://github.com/user-attachments/assets/928af653-8027-4b0e-a406-20ee2dbe506d)
3. The Sentry dashboard is opened, but it reroutes to the main page because the query param is invalid
  a. **Expected**: `https://oneleet.sentry.io/issues/?project=XXXXX&query=id%253A31056c34c88f4332905ff3219e476198`
  b. **Actual**: `https://sentry.io/organizations/Oneleet/issues/?project=XXXXX&query=31056c34c88f4332905ff3219e476198`

## Additional context

It looks like the proper query param format is `&query=id%253A` where `%253A` is the delimiter, and the Sentry ID will follow.

**Comment by pauldambra** [comment]
@silentninja #1338 looks ok as a direct fix for this... but the sentry link works for me in PostHog... we get redirected to the specific event page within sentry.

If we change it as suggested here or in your PR then we land on a search page for the event id in the link

So, they're not the same _really_... I'd love to understand why other people aren't being redirected when posthog's usage is

**Comment by pauldambra** [comment]
so we start with `https://sentry.io/organizations/posthog/issues/?project=XXXXXX&query=ad2067bd8f4341c489efae33f8dcd074`
which 302 refirects to `https://posthog.sentry.io/issues/?project=XXXXXX&query=ad2067bd8f4341c489efae33f8dcd074`
which ends up on `https://posthog.sentry.io/issues/5066165953/events/ad2067bd8f4341c489efae33f8dcd074/?project=XXXXXX&referrer=issue-list&statsPeriod=1h`

**Comment by silentninja** [comment]
@pauldambra, you're correct. I've confirmed Sentry's behavior matches your description. While I previously encountered the issue outlined in this ticket, I'm unable to reproduce it currently. I'll close this PR for now. We can reopen it if someone else experiences the same problem.

**Comment by pauldambra** [comment]
Thanks @silentninja. Really appreciate you taking the time to open the PR 💖
